py/test/selenium/webdriver/firefox/ff_installs_addons_tests.py

```diff
@@ -20,13 +20,19 @@
 from selenium.common.exceptions import WebDriverException
 
 
-def test_install_addon(driver):
+def test_install_addon_temporary(driver):
     extension = os.path.join(os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
                              '../../../../third_party/firebug/favourite_colour-1.1-an+fx.xpi')
 
-    id = driver.install_addon(extension)
+    id = driver.install_addon(extension, True)
     assert id == 'favourite-colour-examples@mozilla.org'
 
+def test_install_addon(driver):
+    extension = os.path.join(os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+                             '../../../../third_party/firebug/favourite_colour-1.1-an+fx.xpi')
```

Comment:
It would be better to add the code below to `/py/BUILD.bazel` and then you can remove most of the path code here
```python
# add ":favourite_color" to the list at line 345

# Goes with the other copy statements
copy_file(
    name = "favourite_color",
    src = "//third_party/favourite_colour-1.1-an+fx.xpi",
    out = "test/selenium/webdriver/firefox/favourite_colour-1.1-an+fx.xpi",
)
```